pkgs/applications/audio/elektroid/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, zlib
+, libzip
+, gtk3
+, alsa-lib
+, json-glib
+, libpulseaudio
+, libsndfile
+, libsamplerate
+}:
+
+stdenv.mkDerivation rec {
+  name = "elektroid";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "dagargo";
+    repo = "elektroid";
+    rev = "${version}";
+    sha256 = "sha256-wpPHcrlCX7RD/TGH2Xrth+oCg98gMm035tfTBV70P+Y=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    zlib
+    libzip
+    gtk3
+    alsa-lib
+    json-glib
+    libpulseaudio
+    libsndfile
+    libsamplerate
+  ];
```

Comment:
Libs should be added into `buildInputs`.

Reply:
Thanks! I've moved everything but the make-type stuff into `buildInputs`, but since I could run the program before the change I don't know how to assess whether the new lists are correct.